packages/sign-client/src/controllers/engine.ts

```diff
@@ -200,6 +200,7 @@ export class Engine extends IEngine {
         metadata: this.client.metadata,
       },
       expiryTimestamp,
+      pairingTopic: topic,
```

Comment:
How does this plays out when a client has an existing `pairingTopic` ?
Is it also handled on the dapp side ?